example/loop_advection/loop_advection_package.cpp

```diff
@@ -0,0 +1,368 @@
+//========================================================================================
+// (C) (or copyright) 2020-2021. Triad National Security, LLC. All rights reserved.
+//
+// This program was produced under U.S. Government contract 89233218CNA000001 for Los
+// Alamos National Laboratory (LANL), which is operated by Triad National Security, LLC
+// for the U.S. Department of Energy/National Nuclear Security Administration. All rights
+// in the program are reserved by Triad National Security, LLC, and the U.S. Department
+// of Energy/National Nuclear Security Administration. The Government is granted for
+// itself and others acting on its behalf a nonexclusive, paid-up, irrevocable worldwide
+// license in this material to reproduce, prepare derivative works, distribute copies to
+// the public, perform publicly and display publicly, and to permit others to do so.
+//========================================================================================
+
+#include "loop_advection.hpp"
+
+#include <iostream>
+#include <utility>
+#include <vector>
+
+//#include "parthenon_mpi.hpp"
+//#include "parthenon_manager.hpp"
+
+using parthenon::DriverStatus;
+using parthenon::MeshBlock;
+using parthenon::Metadata;
+using parthenon::Packages_t;
+using parthenon::ParameterInput;
+using parthenon::Params;
+using parthenon::Real;
+using parthenon::StateDescriptor;
+using parthenon::TaskID;
+using parthenon::TaskList;
+using parthenon::TaskStatus;
+using parthenon::CartDir;
+
+namespace loop_advection_example {
+
+
+struct B_face : public parthenon::variable_names::base_t<false> {
+  template <class... Ts>
+  KOKKOS_INLINE_FUNCTION B_face(Ts &&...args)
+      : parthenon::variable_names::base_t<false>(std::forward<Ts>(args)...) {}
+  static std::string name() { return "B_face"; }
+};
```

Comment:
This looks like the right approach to creating `variable_name` types, although I think we may want to create a macro to reduce boiler plate at some point.

Reply:
I thought that too as I wrote these. Fortunately, I'm guessing defining `variable_name`s in downstream codes will be fairly constrained so there won't be too much boilerplate.